server/metadata/collection.go

```diff
@@ -55,6 +57,72 @@ func (c *CollectionSubspace) getKey(nsID uint32, dbID uint32, name string) keys.
 	return keys.NewKey(c.SubspaceName, c.KeyVersion, UInt32ToByte(nsID), UInt32ToByte(dbID), collectionKey, name, keyEnd)
 }
 
+func (c *CollectionSubspace) Create(ctx context.Context, tx transaction.Tx, nsID uint32, dbID uint32, name string, id uint32, indexes []*schema.Index,
+) (*CollectionMetadata, error) {
+	for _, index := range indexes {
+		index.State = schema.INDEX_ACTIVE
```

Comment:
nit: add a comment that these are indexes created at collection creation time so they are set as active.